tests/src/test/java/Api/AfiInteractiveOptimizerEngineApiTests.java

```diff
@@ -55,6 +57,8 @@ public void enginesApiGetOptimizationSuccess() throws ApiException, JsonProcessi
             AFIOptimizationParameters optimizationUnit = createUnitOptimization();
             AFIOptimizationParametersRoot afiOptimizerParam = new AFIOptimizationParametersRoot();
             afiOptimizerParam.setData(optimizationUnit);
+            Gson gson = new Gson();
+            System.out.println("afiOptimizerParam: " + gson.toJson(afiOptimizerParam));
```

Comment:
Shall we remove these two lines? Also, our library is dependant on `Jackson` and so, I feel that we shouldn't go ahead with `gson`. Can you also remove `gson` dependency in pom.xml?

Reply:
done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/266e7b0894d01ab176698bb1d92877339de4456d